Makefile

```diff
@@ -1,19 +1,22 @@
-prepare-check:
-	ansible-playbook -b -K -i inventory-local -c local ./recipe-tests.yaml
+CONTAINER_NAME=packit
+CONTAINER_EXEC=podman exec $(CONTAINER_NAME)
 
-check:
-	tox
+test_container: test_container_remove
+	rsync -a $(CURDIR)/ /tmp/packit
+	podman build --tag $(CONTAINER_NAME) .
+	podman run --name $(CONTAINER_NAME) -ti -d -v /tmp/packit:/src:Z $(CONTAINER_NAME)
+	sleep 2
 
-# build-tests: recipe-tests.yaml
-# 	ansible-bender build -- ./recipe-tests.yaml $(SOURCE_GIT_IMAGE) $(PACKIT_TESTS_IMAGE)
+test_container_remove:
+	podman stop $(CONTAINER_NAME) || true
+	podman rm -f $(CONTAINER_NAME) || true
+	podman image rm $(CONTAINER_NAME) || true
 
-# shell:
-# 	podman run --rm -ti -v $(CURDIR):/src:Z -w /src $(PACKIT_TESTS_IMAGE) bash
+install:
+	pip3 install .
```

Comment:
```suggestion
	pip3 install --user .
```

Reply:
@TomasTomecek It is because ``pytest`` was not able to find module packit.cli. without installing inside system so that functional tests failed. I've tried variant with ``--user`` and also set ``PYTHONPATH=.`` and it does not work well. That's why I've used this. and also this is why I install target as part of check target, because it invoke installation of changes

Reply:
But that won't work on my laptop: my user can't touch /usr

Reply:
okay. I'll fix it just in testing in container